node/actors/bft/src/lib.rs

```diff
@@ -65,15 +66,28 @@ impl Config {
         mut pipe: ActorPipe<InputMessage, OutputMessage>,
     ) -> anyhow::Result<()> {
         let cfg = Arc::new(self);
+        let leader = leader::StateMachine::new(ctx, cfg.clone(), pipe.send.clone());
+        let replica = replica::StateMachine::start(ctx, cfg.clone(), pipe.send.clone()).await?;
+
+        let (leader_send, leader_recv) =
+            sync::prunable_queue::new(Box::new(leader::StateMachine::queue_pruning_predicate));
+
+        let (replica_send, replica_recv) =
+            sync::prunable_queue::new(Box::new(replica::StateMachine::queue_pruning_predicate));
```

Comment:
Nit: Couldn't this be inside the `new` methods?

Reply:
Fixed, changes here: https://github.com/matter-labs/era-consensus/pull/56/commits/0ac07073f1481289df1d7f6dbbaf31c15f10cd45